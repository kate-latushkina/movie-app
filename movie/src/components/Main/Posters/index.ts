import Posters from "./PosterContainer";

export default Posters;
