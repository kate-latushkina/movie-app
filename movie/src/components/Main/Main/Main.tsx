import { Grid } from "@material-ui/core";
import React from "react";
import styled from "styled-components";
import Categories from "../Categories/Categories";
import Posters from "../Posters/Posters";
import Sorting from "../Sorting/Sorting";

const Main: React.FC = () => {
  return (
    <>
      <CategoryContainer>
        <Categories />
        <Sorting />
      </CategoryContainer>
      <Posters />
    </>
  );
};

const CategoryContainer = styled(Grid)`
  display: flex;
  justify-content: space-between;
  padding 0 15%;
`;

export default Main;
