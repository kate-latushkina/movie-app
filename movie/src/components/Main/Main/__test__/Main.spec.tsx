import React from "react";
import renderer from "react-test-renderer";
import Main from "../Main";

describe("should render Main component", () => {
  it("should render Main snapshot", () => {
    const wrapper = renderer.create(<Main />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
