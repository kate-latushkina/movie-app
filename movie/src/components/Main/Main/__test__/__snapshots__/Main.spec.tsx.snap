// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Main component should render Main snapshot 1`] = `
Array [
  <div
    className="MuiGrid-root sc-eCssSg inwWph"
  >
    <div
      className="MuiTabs-root categories"
    >
      <div
        className="MuiTabs-scroller MuiTabs-fixed"
        onScroll={[Function]}
        style={
          Object {
            "marginBottom": null,
            "overflow": "hidden",
          }
        }
      >
        <div
          className="MuiTabs-flexContainer"
          onKeyDown={[Function]}
          role="tablist"
        >
          <button
            aria-selected={true}
            className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary makeStyles-root-1 Mui-selected"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="tab"
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiTab-wrapper"
            >
              All
            </span>
            <span
              className="PrivateTabIndicator-root-2 PrivateTabIndicator-colorPrimary-3 MuiTabs-indicator"
              style={Object {}}
            />
          </button>
          <button
            aria-selected={false}
            className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary makeStyles-root-1"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="tab"
            tabIndex={-1}
            type="button"
          >
            <span
              className="MuiTab-wrapper"
            >
              Documentary
            </span>
          </button>
          <button
            aria-selected={false}
            className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary makeStyles-root-1"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="tab"
            tabIndex={-1}
            type="button"
          >
            <span
              className="MuiTab-wrapper"
            >
              Comedy
            </span>
          </button>
          <button
            aria-selected={false}
            className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary makeStyles-root-1"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="tab"
            tabIndex={-1}
            type="button"
          >
            <span
              className="MuiTab-wrapper"
            >
              Horror
            </span>
          </button>
          <button
            aria-selected={false}
            className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary makeStyles-root-1"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="tab"
            tabIndex={-1}
            type="button"
          >
            <span
              className="MuiTab-wrapper"
            >
              Crime
            </span>
          </button>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root sc-hKgILt RBNFr"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root sc-gsTCUz lpoOIX MuiInputLabel-animated"
        id="select-label"
      >
        Sort by
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline makeStyles-select-6"
        onClick={[Function]}
      >
        <div
          aria-haspopup="listbox"
          className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
          onBlur={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          role="button"
          tabIndex={0}
        >
          Release date
        </div>
        <input
          aria-hidden={true}
          className="MuiSelect-nativeInput"
          onAnimationStart={[Function]}
          onChange={[Function]}
          tabIndex={-1}
          value="Release date"
        />
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
    </div>
  </div>,
  <div>
    50 movie found
  </div>,
  <div
    className="MuiGrid-root sc-jSgupP dhSJG"
  />,
]
`;
